tests/test_wordfreq_build.py

```diff
@@ -7,28 +7,41 @@
     run_snakemake,
 )
 
+RESULT_EXTRACTED = 'tests/data/extracted'
+RESULT_TOKENIZED = 'tests/data/tokenized'
+RESULT_MESSY_COUNTS = 'tests/data/messy-counts'
+RESULT_COUNTS = 'tests/data/counts'
+RESULT_FREQS = 'tests/data/freqs'
+RESULT_WORDFREQ = 'tests/data/wordfreq'
+REFERENCE_WORDFREQ = 'tests/reference/wordfreq'
+REFERENCE_TOKENIZED = 'tests/reference/tokenized'
+REFERENCE_EXTRACTED = 'tests/reference/extracted'
+REFERENCE_FREQS = 'tests/reference/freqs'
+REFERENCE_COUNTS = 'tests/reference/counts'
+REFERENCE_MESSY_COUNTS = 'tests/reference/messy-counts'
+
 
 @pytest.fixture(scope='session')
 def run_build(test_env_variables, setup_input_files):
     run_snakemake(test_env_variables)
     yield
-    rmtree('tests/data/extracted')
-    rmtree('tests/data/tokenized')
-    rmtree('tests/data/counts')
-    rmtree('tests/data/messy-counts')
-    rmtree('tests/data/freqs')
-    rmtree('tests/data/wordfreq')
+    rmtree(RESULT_EXTRACTED)
```

Comment:
This isn't quite what I meant -- I meant the names ['extracted', 'tokenized', 'messy-counts', 'counts', 'freqs', 'wordfreq'] should be in a list that you can loop over in each place you need to do something to those 6 directories.